libvast/vast/system/actors.hpp

```diff
@@ -200,8 +200,8 @@ using meta_index_actor = typed_actor_fwd<
   // Erase a single partition synopsis.
   caf::replies_to<atom::erase, uuid>::with<atom::ok>,
   // Evaluate the expression.
-  caf::replies_to<expression>::with< //
-    std::vector<uuid>>>::unwrap;
+  caf::replies_to<atom::candidates, vast::expression,
```

Comment:
There is nothing to disambiguate, so why the atom?

Reply:
As discussed offline, I'd like to eventually get *all* handlers to use an atom, since that can be thought of as the "name" of the handler and it enables easy searching for all call sites and all implementations of this interface.